### PR TITLE
Improve the text of some labels

### PR DESCRIPTION
**Pull request by sindresorhus**
## Proposed Changes

It's common to use title-case for status messages like this. Also improved the consistency.

### CODE REVIEW

File: packages/web/src/fixtures/dev-for-apps/functions/useUploadFile.ts [hunk 1]
```diff
@@ -51,22 +51,22 @@ export const useDeleteFile = (accountAddress: string) => {
...
     if (!signature || !signedMessage) {
       return
     }
 
     setIsLoading(true)
-    message.loading({ content: 'delete file...', duration: 0, key })
+    message.loading({ content: 'Deleting File...', duration: 0, key })
 
     const res = deleteUploadFile(signedMessage, signature, accountAddress, id)
       .then(result => {
```
Comment (sindresorhus) [inline packages/web/src/fixtures/dev-for-apps/functions/useUploadFile.ts:64]: Unrelated, but any reasons you're not using async/await? Would make the code much more readable.

Comment (aggre) [reply packages/web/src/fixtures/dev-for-apps/functions/useUploadFile.ts:64]: Thanks for the PR!

By the way, there should be no particular reason not to use async/await here. We can refactor it. Thanks for pointing that out!

**Review by aggre — APPROVED** [review]
LGTM!

File: packages/web/src/fixtures/dev-for-apps/functions/useUploadFile.ts (hunk 1 above)
Comment (sindresorhus) [reply packages/web/src/fixtures/dev-for-apps/functions/useUploadFile.ts:64]: I would recommend the https://github.com/xjamundx/eslint-plugin-promise/blob/development/docs/rules/prefer-await-to-then.md rule. It lets you automate the enforcement.

**Pull request merged**